ChatApp/Janus/Janus.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  ChatApp
+//
+//  Created by Kalvin Dela Raga on 1/27/21.
+//
+
+import UIKit
+import Firebase
+
+class Janus {
+    
+    let window: UIWindow
+    
+    private var listener: AuthStateDidChangeListenerHandle?
```

Comment:
```suggestion
    private var listener: AuthStateDidChangeListenerHandler?
```

Reply:
This is from Firebase API. Look [here](https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth#addAuthStateListener(com.google.firebase.auth.FirebaseAuth.AuthStateListener)).

Reply:
I see this is my bad haven't touched firebase in a long time :joy:

Reply:
Still there codebase needs some refactoring 😂